### DISCUSSION TITLE
How I could retrieve tweet meta data for already collected tweet?

### DISCUSSION THREAD

**Discussion by Elizabithi1** [tweepy/tweepy#1266] [Questions] [answered]
I have collected tweet text in csv file, then I had to extract other features from tweet. could I find any way to do that automatically in Twitter API?. I only have the text of the tweet and I can not go to collect again because of the time limitation of the hashtags. 
please help

**Comment by okomarov** [comment]
You will still need to refetch the tweets with Tweepy. If you have the tweet ids you can refetch exactly those tweets and extract the meta information.

You need to provide more context as it’s hard to help beyond that with this little information.

**Comment by Harmon758** [comment, marked as answer]
As explained, if you have the ID of the Tweet, you can simply use [`API.get_status`](https://tweepy.readthedocs.io/en/latest/api.html#API.get_status).
Otherwise, there's not much you can do with only the text of the Tweet.

**Comment by AchintyaX** [comment]
One has to use  `tweet= api.get_status(tweet_id)` to retrieve a status object and then you can use  `tweet.text` to get the text saved in the tweet
